stdlib/Artifacts/test/runtests.jl

```diff
@@ -263,7 +263,5 @@ end
 end
 
 @testset "Docstrings" begin
-    undoc = Docs.undocumented_names(Artifacts)
     @test_broken isempty(undoc)
```

Comment:
This doesn't look right: you are not setting `undoc` anymore here but still try to use it. Also, this test is not broken anymore.

How about this:
```suggestion
    @test isempty(Docs.undocumented_names(Artifacts))
```